src/controllers/director.correspondence.address.controller.ts

```diff
@@ -84,6 +96,23 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
+const verifyUseRegisteredOfficeAddress = (selectedSraAddressChoice: string, companyProfile: CompanyProfile, officerFilingBody: OfficerFiling): boolean => {
```

Comment:
I might be wrong but seems this are added in as new code from main branch merge?

Reply:
quite the opposite IMO, in these changes, are reflected due to another branch ([dact-1363](https://github.com/companieshouse/officer-filing-web/commits/DACT-1363/)) being merged in this, not the `main`. Once the main is merged the change set should not reflect in this PR.

Reply:
Yes milan is correct,  if i have previously pushed a change for another branch, but that branch hasn't yet been merged,  then i sometimes pull that branch into my new branch, so i can use code i did in that previous change,  as was the case as in 1363.  So nothing to worry about,  once that is merged then then won't be seen here.